api/AuthenticatorAssertionResponse.json

```diff
@@ -0,0 +1,208 @@
+{
+  "api": {
+    "AuthenticatorAssertionResponse": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/AuthenticatorAssertionResponse",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": null
+          },
+          "chrome_android": {
+            "version_added": null
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": true
```

Comment:
FF should be "60":

https://groups.google.com/a/fidoalliance.org/forum/#!searchin/fido-dev/jones%7Csort:date/fido-dev/a0FQGCKWsj0/8OlEZ8FTBQAJ

Reply:
added